extra-utils/pastebinit/autobuild/patches/01-platform_linux_deprecated.patch

```diff
@@ -1,21 +1,24 @@
 --- a/pastebinit
```

Comment:
Please name this file in accordance to the Styling Manual - use Git-like formatted patch file naming. Better yet if you actually generated it using `git format-patch`.